This file should not be converted by rename_dates.py

